addons/event/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,87 @@
+---Fields in module 'event'---
+event        / event.event              / auto_confirm (boolean)        : not a function anymore
```

Comment:
You have to write the same value as it was in v10 (the defaut `True` might not be valid): 

https://github.com/odoo/odoo/blob/14f59f4b780fa1e757cfce6a46c62c7a6b00d13e/addons/event/models/event.py#L179